src/Client.php

```diff
@@ -401,6 +402,33 @@ public function send($method, $payload, $path, $params = [], array $headers = []
             return null;
         }
 
+        $responseParsers = [
+            'application/json' => [$this, 'parseJson'],
+            'application/pdf' => [$this, 'parsePdf'],
+        ];
+
+        if (preg_match('/^(\w+\/\w+).*/i', $response->getHeaderLine('Content-Type'), $matches)) {
+            $contentType = $matches[1];
```

Comment:
could it happen that someone uses an uppercase letter? would make sense to use a strtolower()?